Dockerfile.ocp

```diff
@@ -0,0 +1,55 @@
+FROM registry-proxy.engineering.redhat.com/rh-osbs/openshift-golang-builder:rhel_9_golang_1.22 AS build
+
+#ENV GOPATH=/go
+#ENV PATH=$GOPATH/bin:/usr/local/go/bin:$PATH
+
+WORKDIR /workspace
+# Copy the Go Modules manifests
+COPY go.mod go.mod
+COPY go.sum go.sum
+# cache deps before building and copying source so that we don't need to re-download as much
+# and so that source changes don't invalidate our downloaded layer
+RUN go mod download
```

Comment:
I think we can get rid of the go mod stanzas here, since we are vendoring code.

Reply:
We can get rid of the go mod download, but I get errors if go.mod and go.sum are copied over.